packages/shared/scope-hoisting/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@parcel/scope-hoisting",
+  "version": "2.0.0",
+  "description": "Blazing fast, zero configuration web application bundler",
+  "main": "src/index.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/parcel-bundler/parcel.git"
+  },
+  "engines": {
+    "node": ">= 8.0.0"
+  },
+  "scripts": {
+    "pretest": "yarn build",
+    "test": "cross-env NODE_ENV=test mocha",
+    "test-ci": "yarn build && yarn test",
+    "build": "babel src -d lib",
+    "prepublish": "yarn build"
+  },
+  "devDependencies": {
```

Comment:
we shouldn't need these `devDependencies` since they're listed in the root package.json